hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSPermissionChecker.java

```diff
@@ -549,7 +549,6 @@ private boolean hasPermission(INodeAttributes inode, FsAction access) {
    * - Default entries may be present, but they are ignored during enforcement.
    *
    * @param inode INodeAttributes accessed inode
-   * @param snapshotId int snapshot ID
```

Comment:
It seems that is not related to this changes.

Reply:
I just reverted this change.